src/components/App.tsx

```diff
@@ -190,6 +190,31 @@ const App: FC<AppProps> = ({ demoId }) => {
     [currentPathway, evaluatedPathways]
   );
 
+  const assignPathway = useCallback(
+    (pathwayName: string): void => {
+      if (!patient) return;
+
+      const carePlan = createCarePlan(pathwayName, patient);
+
+      setPatientRecords([...patientRecords, carePlan]);
+      client?.create?.(carePlan);
+    },
+    [patientRecords, patient, client, setPatientRecords]
+  );
+
+  const unassignPathway = useCallback(
+    (pathwayName: string): void => {
+      const newPatientRecords: DomainResource[] = [];
+      patientRecords.forEach(r => {
+        if (r.resourceType === 'CarePlan' && (r as CarePlan).title === pathwayName)
+          client?.delete?.(`CarePlan/${r.id}`);
```

Comment:
For discussion only, no action requested: are there potential benefits/drawbacks to deleting the CarePlan vs marking it as `status: "revoked"` or possibly something else?

Reply:
I guess that depends if we are going to be populating this CarePlan with any details. If we are then it would make sense to keep it and just change the status